tensorflow/python/autograph/operators/py_builtins.py

```diff
@@ -342,7 +342,24 @@ def _py_enumerate(s, start=0):
   return enumerate(s, start)
 
 
-SUPPORTED_BUILTINS = (abs, float, int, len, print, range, enumerate)
+def zip_(*s):
```

Comment:
To be as consistent with the Python spec as we can - could you name this argument `iterables`? Thanks!

Reply:
okayy